### PR TITLE
[DO NOT MERGE] Migrate notebook_executor.sh from Slava repo

### PR DESCRIPTION
**Pull request by gogasca**
Did some minor refactors

### CODE REVIEW

File: dlvm/tools/scripts/notebook_executor.sh [hunk 1]
```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (nnegrey) [inline dlvm/tools/scripts/notebook_executor.sh]: Is this the right license?

**Comment by nnegrey** [comment]
What's the context on this PR?

**Comment by gogasca** [comment]
> What's the context on this PR?

We are adding support for GCS in papermill. This PR was a workaround which may be modified in few weeks as we natively support it via papermill gs://bucket/notebook.ipnyb https://blog.kovalevskyi.com/deeplearning-images-revision-m12-papermill-from-netflix-cudnn-7-4-chainer-5-0-0-6e4f87f514ef

File: dlvm/tools/scripts/notebook_executor.sh [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        echo "Installing driver"
+        /opt/deeplearning/install-driver.sh
+      fi
+    fi
+    return 0
+}
+
+
+function metadata_exists() {
+    http_code=$(curl -i http://metadata.google.internal/computeMetadata/v1/instance/attributes/$1 -H "Metadata-Flavor: Google" -o /dev/null -w '%{http_code}\n' -s)
```
Comment (nnegrey) [inline dlvm/tools/scripts/notebook_executor.sh]: `.internal`?
Is that actually internal or not?

Comment (nnegrey) [reply dlvm/tools/scripts/notebook_executor.sh]: Gotcha

**Pull request merged**